src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div
+        class="card__image"
+        alt="Product image"
+      ></div>
+
+      <div class="card__title">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
```

Comment:
The product code also should be semantically marked up, consider using a `span` or `p` with an additional class for styling instead of nesting paragraphs.